tests/AppBundle/Controller/ClientControllerTest.php

```diff
@@ -121,10 +121,9 @@ public function testupsertPostLayDeputy()
         $this->assertEquals('Firstname', $client->getFirstname());
         $this->assertCount(1, $client->getUsers());
         $this->assertEquals(self::$deputy1->getId(), $client->getUsers()->first()->getId());
-        $this->assertInstanceOf('AppBundle\Entity\Ndr\Ndr', $client->getNdr());
     }
 
-    public function testupsertPut()
+    public function testupsertPut_lay_deputy()
```

Comment:
camelCase?

Reply:
I used to be exclusively camel case for test function names until my last role where we kept the actual function it was testing in camel case but gave the scenario/detail using snake case. Happy to change if thats the preferred style here though!